### PR TITLE
Added repos

### PR DESCRIPTION
**Pull request by mihirk284**
Signed-off-by: Mihir Kulkarni <mihirk284@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,19 @@ Fork this repository and make changes to the forked repository. Make the changes
 #### Talks
   * [The astounding athletic power of quadcopters](https://youtu.be/w2itwFJCgFQ)
 ## Repositories
-* Autonomous landing UAV [[Link]](https://github.com/MikeS96/autonomous_landing_uav)
+
+### Repositories from BITS Pilani
+
+* GenNav ERC-BPGC [[Link]](https://github.com/ERC-BPGC/gennav)
+* Omnibase ERC-BPGC [[Link ](https://github.com/ERC-BPGC/omnibase)]
```
Comment (Matnay) [inline README.md]: minor correction needed in link syntax on line 48

**Comment by hardesh** [comment]
Can you also add these gazebo dataset repos:
https://github.com/clearpathrobotics/cpr_gazebo
https://github.com/tudelft/gazebo_models
https://github.com/mlherd/Dataset-of-Gazebo-Worlds-Models-and-Maps 

It would be better if we add all the repos in a single PR

**Comment by mihirk284** [comment]
> Can you also add these gazebo dataset repos:
> https://github.com/clearpathrobotics/cpr_gazebo
> https://github.com/tudelft/gazebo_models
> https://github.com/mlherd/Dataset-of-Gazebo-Worlds-Models-and-Maps
> 
> It would be better if we add all the repos in a single PR

@hardesh  done

File: README.md (hunk 1 above)
Comment (mihirk284) [reply README.md]: done

**Review by hardesh — APPROVED** [review]
looks good! :)

Btw, if this is going to be like a list of learning resources, then do checkout: https://github.com/IvLabs/resources
@mihirk284 @Matnay

File: README.md (hunk 1 above)
Comment (MihirDharmadhikari) [inline README.md]: One ']' missing after the 'Link' in the Omnibase link

**Review by MihirDharmadhikari — APPROVED** [review]
Looks good. Just a small change that I have mentioned. We should add the Aero handbook also I think.

File: README.md (hunk 1 above)
Comment (mihirk284) [reply README.md]: This is outdated. This is fixed in the latest commit

**Pull request merged**